core/src/View/Hooks/Kokkos_DynamicViewHooks.hpp

```diff
@@ -0,0 +1,141 @@
+#ifndef KOKKOS_EXPERIMENTAL_DYNAMICVIEWHOOKS_HPP
+#define KOKKOS_EXPERIMENTAL_DYNAMICVIEWHOOKS_HPP
+
+#include <Kokkos_ViewHolder.hpp>
```

Comment:
Include what you use here, `<functional>`, `<thread>`, etc.

Reply:
added here but `<mutex>` instead of `<thread>`